liteflow-labs/liteflow-js
# update cli and compiler to switch from ipfs to minio for services' source

update cli and compiler to switch from ipfs to minio for services' source

to test:
- build the backend with PR https://github.com/liteflow-labs/backend/pull/223.
- update in the cli's docker-compose the backend tag
- use command `dev`
- go to localhost:9000 and see the file
